config.yml

```diff
@@ -61,5 +61,5 @@ sum_keys:
 sum_format: '\d+(\.\s?|,\s?|[^a-zA-Z\d])\d{2}'
 
 # Matches dates like 19.08.15 and 19. 08. 2015
-date_format: '.*?(?P<date>(\d{2,4}(\.\s?|[^a-zA-Z\d])\d{2}(\.\s?|[^a-zA-Z\d])(19|20)?\d\d)\s+'
+date_format: '.*?(?P<date>(\d{2,4}(\.\s?|[^a-zA-Z\d])\d{2}(\.\s?|[^a-zA-Z\d])(20)?1[3-6]))\s+'
```

Comment:
I know that it's the original regex but doesn't that just match years from 2013 to 2016? Maybe we can keep the `\d\d` part? What do you think?

Reply:
Ah, yes, sure. I've returned to the old regexp, to test against previously working data/regexp.